### ISSUE TITLE
support for multiple Remote instances

### ISSUE THREAD

**Issue by jderose-net** [custom-components/remote_homeassistant#318]
Hi. I tried adding two Remote instances to Master (via yaml) but it seems only the last instance in the config gets loaded. The documentation seems to indicate this configuration is possible, but before I go too far down this path does anyone know if this integration supports multiple Remote instances? Thanks, JD

**Comment by jaym25** [comment]
@jderose-net Yes, it does support multiple remote instances.  You may have to use YAML to set it up.  The GUI setup is not complete and needs some work...
